src/resolvers/message.ts

```diff
@@ -1,25 +1,76 @@
-import { Message, Resolvers } from '../generated/graphql';
+import { Message, MessagesConnection, Resolvers } from '../generated/graphql';
+import { Op, WhereOptions } from 'sequelize';
 
 import { checkAuth } from '../utils/auth';
+import paginateResults from '../utils/pagination';
 
 const resolver: Resolvers = {
   Query: {
     messages: async (
       _,
-      args, {
-        verifyUser,
-        models,
-      },
-    ): Promise<Message[]> => {
+      args,
+      { verifyUser, models },
+    ): Promise<MessagesConnection> => {
       const { Message: messageModel } = models;
       const auth = verifyUser();
       checkAuth(auth);
 
-      return messageModel.findAll({
-        where: {
-          messageId: 1,
-        },
+      const { message, channelId, pageSize = 20, after } = args;
+      let qryMsg: object = {};
+      let qryAfter: object = {};
+      let qryChannelId: object = {};
+
+      if (message) {
+        const msgOpConds: object = {};
+        const findColName = 'text';
+        msgOpConds[findColName] = {
+          [Op.like]: `%${message}%`,
+        };
+        qryMsg = {
+          [Op.or]: msgOpConds,
+        };
+      }
+      if (after) {
+        qryAfter = {
+          createdAt: { [Op.lt]: Number(after) },
+        };
+      }
+      if (channelId) {
+        qryChannelId = {
+          channelId,
+        };
+      }
```

Comment:
It looks like that this isn't a good code either. I think you should make some guidelines if you want to develop things in your style and tell us why writing like this would be better. Otherwise, you should better follow codes similar to those already written before.

Opensource should be developed as if it is written by one person.

Reply:
I want to make sure that I don't insist on doing my own way and try to follow standard or refrence like you said.
I just referred to similar codes already written before.
- [user.ts#L109: before I hadn't touched](https://github.com/dooboolab/hackatalk-server/blob/1ce4941a2dfd3cbe7344f0e4852267160ceac168/src/resolvers/user.ts#L109).

Reply:
What would be a bit better is try to `return` it if something should be considered refactored. It's ok for producing similar codes for readability in first hand.

Reply:
What's different from the current code is that I've tried to clarify conditions like `when args exits` => `then do this` and tried to clarify what it is doing for each condition.

The flow for yours looks much broad and would take a bit more time to catch up on what it is returning based on the conditions. I hope you've understood my point.

Please bear in mind, I've tried to tackle the problem and improvement that I can see here. I love your challenge and contribution ❤️

Reply:
Sorry if I sound ungrateful here. I didn't mean it.